### PR TITLE
feat(tabs-extended): update to v11 styles and deprecate `vertical` variant

### LINKED ISSUE #10756: [Component] Tabs extended
Component [review notes](https://ibm.ent.box.com/notes/1204686003278)

[Tabs extended v2 design specs](https://www.figma.com/file/zFcPEDXLThKHH5dIrUth5S/Tabs-extended-v2?type=design&node-id=10%3A6400&mode=design&t=v23ZMK4zkKKMnmbL-1)

```[tasklist]
### Tasks
- [x] Update tabs extended to use carbon wc tabs – but 16px type size
- [x] Use v11 type styles
- [x] Use v11 color styles
- [x] Deprecate `vertical` variant
- [x] Storybook demo option with pre populated content item horizontal (?)
```

### PR DESCRIPTION
**Pull request by emyarod**
### Related Ticket(s)

Closes #10756

### Description

This PR updates the tabs-extended component to use v11 Carbon styles and deprecates the `vertical` tabs variant

### CODE REVIEW

File: packages/web-components/src/components/tabs-extended-media/__stories__/tabs-extended-media.stories.ts [hunk 1]
```diff
@@ -8,121 +8,160 @@
...
-                <c4d-content-item-row-media-video
-                  video-id="0_ibuqxqbe"></c4d-content-item-row-media-video>
-              `
-            : ''}
-          <c4d-content-item-heading>Tab heading ${i}</c4d-content-item-heading>
-          <c4d-content-item-row-media-copy
-            >Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean
-            et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus
-            at elit sollicitudin.</c4d-content-item-row-media-copy
+        <c4d-card-link-cta slot="footer" href="https://example.com">
```
Comment (IgnacioBecerra) [inline packages/web-components/src/components/tabs-extended-media/__stories__/tabs-extended-media.stories.ts:57]: This one should also be `<c4d card link>`

**Comment by ariellalgilmore** [comment]
I noticed the same thing @IgnacioBecerra with the tab titles... seems like an issue with `cds-tab`. Note on [tab-width](https://react.carbondesignsystem.com/?path=/docs/components-tabs--overview#max-width). 

If we remove the the `max-width` it should work. Guessing maybe design might want to shorten the storybook copy. But it would be nice to make this change in cwc-v2 first
Screenshot 2023-11-16 at 8 05 11 AM

**Comment by emyarod** [comment]
I think the card-link and card-footer elements should be correct in the story now!

for the tab title ellipses, I just had to remove the display: flex rule for the ellipses to show up properly

**Review by IgnacioBecerra — APPROVED** [review]
LGTM, thanks @emyarod!

**Comment by ariellalgilmore** [comment]
really closee!!! I think the contained tabs height is now too small (40px) instead of (48px). Looks like it's just missing these styles, which were added for `cds-tabs`,  but need to also be added for `<c4d-tabs-extended type="contained"/>`  https://github.com/emyarod/carbon-for-ibm-dotcom/blob/10756-feat/tabs-v2/packages/carbon-web-components/src/components/tabs/tabs.scss#L123-L138

48px height
Screenshot 2023-11-16 at 5 24 36 PM

**Review by oliviaflory — APPROVED** [review]
@emyarod yesss tabs looks great! 

woohoo

**Pull request merged**